File: src/main/java/br/com/dusty/dkits/ability/Abilities.java
package br.com.dusty.dkits.ability;

import br.com.dusty.dkits.Main;
import org.bukkit.Bukkit;
import org.bukkit.plugin.PluginManager;

import java.util.ArrayList;

public class Abilities {
	
	public static final ArrayList<Ability> ABILITIES = new ArrayList<>();
	
	public static void registerAll() {
		//Usage: ABILITIES.add(FOO_ABILITY);
		
		PluginManager pluginManager = Bukkit.getPluginManager();
		ABILITIES.forEach(ability -> pluginManager.registerEvents(ability, Main.INSTANCE));
	}
}

File: src/main/kotlin/br/com/dusty/dkits/ability/Abilities.kt
package br.com.dusty.dkits.ability

import br.com.dusty.dkits.Main
import org.bukkit.Bukkit

object Abilities {

	val ABILITIES = hashSetOf<Ability>()

	fun registerAll() {
		//Usage: ABILITIES.add(FOO);

		ABILITIES.add(BubbleAbility)
		ABILITIES.add(ChemistAbility)
		ABILITIES.add(ChickenmanAbility)
		ABILITIES.add(ElfoAbility)
		ABILITIES.add(EndermageAbility)
		ABILITIES.add(FishermanAbility)
		ABILITIES.add(GladiatorAbility)
		ABILITIES.add(KangarooAbility)
		ABILITIES.add(NinjaAbility)
		ABILITIES.add(OdinAbility)
		ABILITIES.add(PoseidonAbility)
		ABILITIES.add(RingAbility)
		ABILITIES.add(SnailAbility)
		ABILITIES.add(StomperAbility)
		ABILITIES.add(ThorAbility)
		ABILITIES.add(TurtleAbility)
		ABILITIES.add(ViperAbility)

		ABILITIES.forEach { ability -> Bukkit.getPluginManager().registerEvents(ability, Main.INSTANCE) }
	}
}
